scripting/get5/teamlogic.sp

```diff
@@ -1,16 +1,26 @@
 public Action Command_JoinGame(int client, const char[] command, int argc) {
-  if (g_GameState == Get5State_None || !IsPlayer(client)) {
-    return Plugin_Continue;
+  // We are required to put a FakeClientCommand on a timer, 
```

Comment:
I think it's good now, just clean up this comment, as there is now no `FakeClientCommand` call.

Suggest:

1. Add a check for g_CheckAuthsCvar + gamestate == none here, so we don't start timers every time a user joins if Get5 is not doing any team logic anyway.
2. Change comment to something like:
```
In order to avoid duplication of team-join logic, we directly call the same handle that would be called
if the user selected any team after joining. Since Command_JoinTeam handles the actual joining using a
FakeClientCommand, we don't have to do any team-logic here and it won't matter what we pass to Command_JoinTeam. The only thing that's important is that the command argument is empty, as that avoids a call to GetCmdArg in that function.
```